apps/cowswap-frontend/src/modules/ethFlow/containers/EthFlow/EthFlowBanner.tsx

```
@@ -22,17 +22,16 @@ export interface EthFlowBannerProps extends EthFlowBannerCallbacks {
 export function EthFlowBanner({ hasEnoughWrappedBalance, ...props }: EthFlowBannerProps) {
   const [showBanner, setShowBanner] = useState(false)
   const isNativeIn = useIsNativeIn()
+  const { isReady } = useTradeStateReadiness()
   const native = useNativeCurrency()
   const wrapped = useWrappedToken()
 
-  // TODO: Add proper return type annotation
-  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-  const showBannerCallback = () => {
+  const showBannerCallback = (): void => {
     return setShowBanner((state) => !state)
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)